Plugin/CleverTapUnity/CleverTapUnitySDK/CleverTapUnityExample.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using UnityEngine;
+using System.Collections.Generic;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace CleverTapUnitySDK {
+    public class CleverTapUnityExample : MonoBehaviour {
```

Comment:
This is not needed for now, as we are still using the `CleverTapUnity`. Later we will add Sample and HowTo use the SDK

Reply:
Removed

Reply:
We will need a sample and how-to before the release of those changes. It can be done outside this PR but it will be required before the release since current docs point to the obsolete methods.